README.md

```diff
@@ -1 +1,63 @@
-# better-unit
\ No newline at end of file
+# better-unit
+
+better-unit is a wrapper around the [nodeunit](https://www.npmjs.com/package/nodeunit) module, with the following additions:
+
+* Support for comparing `Map`, `Set`, `WeakMap`, and `WeakSet` using the `deepEqual()` assertion.
```

Comment:
nodeunit reproduces node's built-in assertions, and node's assert module is locked. So it's likely impossible to get this into either.

We can certainly build plugins for the individual pieces, but unless we have a compelling reason to, I'd rather not have multiple repos to maintain.